pkg/inspecttypes/dockercompat/dockercompat.go

```diff
@@ -214,9 +224,21 @@ func networkSettingsFromNative(n *native.NetNS) *NetworkSettings {
 		// TODO: set CNI name when possible
 		fakeDockerNetworkName := fmt.Sprintf("unknown-%s", x.Name)
 		res.Networks[fakeDockerNetworkName] = nes
+
+		var ports []gocni.PortMapping
+		err := json.Unmarshal([]byte(sp.Annotations[labels.Ports]), &ports)
```

Comment:
Skip setting `res.Ports` when `if portsLabel, ok := sp.Annotations[labels.Ports]; !ok`

Reply:
done with omit the empty json. is it for you ?